modules/ROOT/pages/asset-types.adoc

```diff
@@ -6,23 +6,56 @@ endif::[]
 
 The following asset types are supported in Exchange:
 
-* Connectors: Packaged connectivity to an Anypoint Platform endpoint using third-party APIs and standard integration protocols
+* API Groups
++
+A set of APIs bundled into a single asset.
+* API Spec Fragments
++
+A RAML document that has a version and an identifier but is not in itself a complete RAML specification.
++
+API spec fragments are also known as _RAML fragments_.
+* AsyncAPI Specifications
++
+An AsyncAPI specification file that specifies an event-driven API
+* Connectors
++
+Packaged connectivity to an Anypoint Platform endpoint using third-party APIs and standard integration protocols.
 +
 Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.
-* Templates: Packaged integration patterns built on best practices to address common use cases
+
+* Custom
 +
-Complete the template's use case or solution by supplying your own information, customizing or extending the templates as needed.
-* Examples: Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution
-* Policies: Configuration modules to extend the functionality of an API and enforce capabilities such as security
-* Ruleset: YAML files that describe a ruleset
-* REST APIs: RAML or OAS files that specify APIs referenced by an HTTP Request connector to expose metadata to Anypoint Studio
-* AsyncAPI specifications: An AsyncAPI specification file that specifies an event-driven API
-* SOAP APIs: A WSDL file that specifies an API
-* HTTP APIs: A placeholder for an endpoint for use by private Exchange users who want to manage the endpoint with API Manager
-* API Groups: A set of APIs bundled into a single asset
+A description and an optional file to explain aspects of your system, to provide instructional videos, or to describe product or organizational documentation.
+* Examples
++
+Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution.
+* GraphQL API 
++
+A schema definition that contains the object and types definitions that are used to interact with the API through the GraphQL specification. GraphQL enables you to query an API that supports this language in a much more flexible way than REST.
```

Comment:
should this be _objects and types_ or _object and type_ ?

Reply:
should be ... object types and definitions...